srcpkgs/xarcan/template

```diff
@@ -1,36 +1,22 @@
 # Template file for 'xarcan'
 pkgname=xarcan
-version=0.5.4
-revision=8
-build_style=gnu-configure
+version=0.6.0
+revision=1
+build_style=meson
+configure_args="-Dglamor=true -Dint10=false -Dipv6=true -Dsystemd_logind=false
+ -Ddefault_font_path=/usr/share/fonts/X11"
```

Comment:
Can it use elogind, perhaps?

Reply:
No elogind support, it has its own logind like behaviour (it splits off into a privileged device-control process that act as watchdog, cleans up state on crashes etc. then the normal-user engine).

Reply:
Got it. As an alternative, you could look into seatd/libseat as well, since it _might_ be simpler to support multiple platforms that way.

Reply:
well, multiple but not for Open/NetBSD and at the cost of important features and safety - I am not strictly against an alternative implementation to the interface in psep_open that uses it, but I won't go near it myself.